libvast/test/flow.cpp

```diff
@@ -0,0 +1,68 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#define SUITE flow
+
+#include "vast/flow.hpp"
+
+#include "vast/test/test.hpp"
+
+#include "vast/address.hpp"
+#include "vast/concept/parseable/to.hpp"
+#include "vast/concept/parseable/vast/address.hpp"
+#include "vast/concept/parseable/vast/port.hpp"
+#include "vast/port.hpp"
+
+using namespace vast;
+
+namespace {
+
+struct fixture {
+  flow tcp_flow = flow{unbox(to<address>("10.10.0.1")),
+                       unbox(to<address>("10.10.0.2")),
+                       unbox(to<port>("123/tcp")), unbox(to<port>("321/tcp"))};
+
+  flow udp_flow = flow{unbox(to<address>("10.10.0.1")),
+                       unbox(to<address>("10.10.0.2")),
+                       unbox(to<port>("123/udp")), unbox(to<port>("321/udp"))};
+};
```

Comment:
Is there a reason why ICMP and ICMP6 are not part of the fixture?

Reply:
The purpose of these two test flows is to have flows that differ only in their port type. Adding more protocols would not result in more coverage and only add redundant checks.